app/components/radio_button/radio_button_group.js

```diff
@@ -17,32 +17,6 @@ export default class RadioButtonGroup extends PureComponent {
         options: [],
     };
 
-    constructor(props) {
-        super(props);
-
-        this.state = {
-            selected: this.getSelectedValue(props.options),
-        };
-    }
-
-    componentWillReceiveProps(nextProps) {
-        if (this.props.options !== nextProps.options) {
-            this.setState({selected: this.getSelectedValue(nextProps.options)});
-        }
-    }
-
-    getSelectedValue = (options = []) => {
-        let selected;
-        for (const option in options) {
-            if (option.checked) {
-                selected = option.value;
-                break;
-            }
-        }
-
-        return selected;
-    }
-
     onChange = (value) => {
         const {onSelect} = this.props;
         this.setState({
```

Comment:
after you last change, do you need this?

Reply:
You are right.  I don't need this.setState.

Reply:
Done.